libs/constrained-generators/src/Constrained/Test.hs

```diff
@@ -152,6 +152,9 @@ tests =
     , testSpec "listSumPair" (listSumPair @Int)
     , testSpec "parallelLetPair" parallelLetPair
     , numberyTests
+    , testSpec "(And pair (x,y) test -- both (x <=. 5) and (y <=. 1)" andPair
+    , testSpec "(Or  pair (x,y) test -- at least one of (x <=. 5) or (y <=. 5)" orPair
```

Comment:
I would prefer we rename these two tests to follow the naming scheme of the rest of the file.

However, I like the idea of having information about what the test contains be reported on failure. I'll add a `counterexample` for the `spec` to the `prop_sound` property so that way we get the same effect there.

Reply:
done